docs/native-modules-setup.md

```diff
@@ -222,7 +242,7 @@ If you are working on an existing module that already has iOS and Android sample
 4. Open the solution with Visual Studio and [link native module](native-modules-using.md).
 > The project should build correctly at this point, but we still need to setup some special metro configurations for Windows in order to run the app without breaking iOS and Android bundling.
 
-5. Add `metro.config.windows` for Windows bundling ([example](https://github.com/react-native-community/react-native-webview/blob/master/metro.config.windows.js)).
+5. Add `metro.config.windows` for Windows bundling ([example](https://github.com/react-native-community/react-native-webview/blob/master/metro.config.windows.js)). Make sure the config file is at the root of the repo (see [Metro bug #588](https://github.com/facebook/metro/issues/588)). 

```

Comment:
If we have 0.62 be the min version, then we shouldn't need to do anything special for metro.config.  -- I think I'd rather keep the docs as only whats needed to support >=0.62, and remove all the logic about metro.config.

Reply:
Unfortuntely the part about not adding metro.config.js logic is not true, as we have windows specific behavior like react-native bundle failing when being invoked from msbuild because msbuild writes to ProjectImports.zip, see #5050.